Refactor: Create a separate test for the commented-out expectation to ensure it’s recognized by the engine.

MojtabaHs opened the pull request:
This test now compiles but is isolated to ensure it doesn’t interfere with the existing test plan.

MojtabaHs committed:
test: refactor the tests and extract the disabled condition

amosavian commented:
This expectation is to ensure equality of JWKS works regardless of key order. This is commented out because it does not work rright now.

MojtabaHs commented:
Here is my intention: 
- Extracted the commented-out code because it was not being used yet.
- Added it as a separate test to highlight its importance while keeping the original implementation intact.
- Disabled the test since it is not functioning correctly at the moment.

Would you like to rename the test case to better represent it? How about writing a dedicated integrated case containing both of these?
